frontend/packages/knative-plugin/src/components/add/event-sources/EventSourcesSelector.tsx

```diff
@@ -25,6 +37,8 @@ const EventSourcesSelector: React.FC<EventSourcesSelectorProps> = ({ eventSource
     setErrors,
     setStatus,
   } = useFormikContext<FormikValues>();
+  const [filteredSources, setFilteredSources] = React.useState(eventSourceList);
+  const [filterText, setFilterText] = React.useState('');
```

Comment:
Why do we need this state for `filterText`? State for `filteredSources` should be enough.

Reply:
State for `filterText` is used to clear the input field when "clear all filters" is clicked from the empty state.